packages/btc/src/ckb/commitment.ts

```diff
@@ -0,0 +1,60 @@
+import { sha256 } from 'js-sha256';
```

Comment:
I think the `ckb` module which includes `commitment.ts`, `molecule.ts`, and `rpc.ts` should be removed from the BTC package. Any method about CKB should be implemented in CKB package and is called by BTC package. 

If the CKB package has no related methods, I will add them ASAP.

Reply:
Okay, for that I think I need the following APIs: 
- calculateCommitment: calculate commitment from RawTransaction (provided)
- isRgbppLockCell: check if an input (or cell, see what you can do to simplify things) is an RgbppLock cell
- isRgbppTimeLockCell: similar logic as `isRgbppLockCell`, checks if an input/cell is an RgbppTimeLock cell
- getCellByOutPoint: Query a cell by an OutPoint, do the querying work, but if the above methods accepts `Input` instead of `Cell`, this method is redundant

Reply:
Added in https://github.com/ckb-cell/rgbpp-sdk/pull/34

Reply:
Refactored, removed most ckb deps, leaving a few types from lumos: https://github.com/ckb-cell/rgbpp-sdk/pull/26/commits/e630f922b208bae3edc63310404eb6714759a8ce

Reply:
Replaced lock checking with cell output checking: https://github.com/ckb-cell/rgbpp-sdk/pull/26/commits/b93177be98021874bb7be6c4a74d3813fafe2954